docs/developer_setup.md

```diff
@@ -0,0 +1,55 @@
+# Developer Setup
+
+Logging in to the SUI requires a running instance of ManageIQ. Instructions on how to install ManageIQ can be found
+[here](https://github.com/ManageIQ/guides/blob/master/developer_setup.md).
+
+### Install Repository and Prerequisites
+
+- Have the [ManageIQ](http://github.com/ManageIQ/manageiq) repo cloned into a
+  directory named `manageiq`, and ready to be started. See [here](https://github.com/ManageIQ/guides/blob/master/developer_setup.md)
+  for the steps required to setup ManageIQ.
+- Have this repo as a sibling to the `manageiq` directory:
```

Comment:
Does this need to be cloned inside the manageiq dir?  I don't do that...seems like it "muddies" the waters.   Thoughts?